.buildkite/pipeline.gpu.yml

```diff
@@ -20,6 +20,15 @@
     - bazel test --config=ci $(./ci/run/bazel_export_options) --build_tests_only --jobs 1
       --test_tag_filters=gpu --test_env=RAY_USE_MULTIPROCESSING_CPU_COUNT=1 --test_env=RLLIB_NUM_GPUS=1 rllib/...
 
+- label: ":book: Doctest (GPU)"
```

Comment:
For some reason the GPU step runs all of the `doc_code` modules, but the CPU step doesn't. I'm not why the behavior is different. The only difference is the `+gpu` test tag filter.

Reply:
Ah, figured it out. Needed to mark the `doctest` tag as required.